src/NServiceBus.ContainerTests/When_using_nested_containers.cs

```diff
@@ -148,14 +159,16 @@ public void UoW_components_in_the_parent_container_should_be_singletons_in_the_c
 
                 using (var nestedContainer = builder.BuildChildContainer())
                 {
-                    Assert.AreSame(nestedContainer.Build(typeof(InstancePerUoWComponent)), nestedContainer.Build(typeof(InstancePerUoWComponent)), "UoW's should be singleton in child container");
+                    var instance1 = nestedContainer.Build(typeof(InstancePerUoWComponent));
+                    var instance2 = nestedContainer.Build(typeof(InstancePerUoWComponent));
+
+                    Assert.AreSame(instance1, instance2, "UoW's should be singleton in child container");
                 }
             }
         }
 
         [Test]
-        [Explicit]
-        public void UoW_components_should_by_instance_per_call_in_root_container()
+        public void UoW_components_should_be_singletons_in_root_container()
         {
             using (var builder = TestContainerBuilder.ConstructBuilder())
             {
```

Comment:
Also, I know it's not your code, but what purpose would the noop check be doing? If we're checking for side effects that should be in another explicit test.